### PR TITLE
show progress dialog while downloading..

### PR DESCRIPTION
**Pull request by rrijal53**

### CODE REVIEW

**Comment by aakarnik** [comment]
5.5in-portrait
5.5in-landscape

10in-potrait
10in-landscape

**Comment by rrijal53** [comment]
thanks for the review @leomaxi @aakarnik  i'll fix thore issue ASAP.

File: app/src/main/java/org/ole/planet/takeout/DashboardFragment.java [hunk 1]
```diff
@@ -158,80 +172,100 @@ public void myLibraryDiv(View view) {
...
+                    }).setNeutralButton(R.string.download_all, new DialogInterface.OnClickListener() {
+                        @Override
+                        public void onClick(DialogInterface dialogInterface, int i) {
+                            downloadAllFiles(db_myLibrary);
+                        }
+                    }).setNegativeButton(R.string.txt_cancel, null).show();
         }
     }
 
+//    private void showDownloadDialog() {
```
Comment (leomaxi) [inline app/src/main/java/org/ole/planet/takeout/DashboardFragment.java]: Please clean-up this `showDownloadDialog` block of  code if not in use..

File: app/src/main/java/org/ole/planet/takeout/datamanager/MyDownloadService.java [hunk 2]
```diff
@@ -122,6 +123,7 @@ private void downloadFile(ResponseBody body) throws IOException {
             long currentTime = System.currentTimeMillis() - startTime;
```
Comment (leomaxi) [inline app/src/main/java/org/ole/planet/takeout/datamanager/MyDownloadService.java]: Check and clear unnecessary line breaks / white spaces..

**Comment by rrijal53** [comment]
@aakarnik  
Please review again.
Download dialog is working fine on my 10inch emulator 
screenshot_1531789511

**Pull request merged**